bip39-lib/src/commonMain/kotlin/cash/z/ecc/android/bip39/Mnemonics.kt

```diff
@@ -24,7 +24,9 @@ object Mnemonics {
     const val DEFAULT_LANGUAGE_CODE = "en"
 
     internal val secureRandom = SecureRandom()
-    internal var cachedList = WordList()
+
+    @Suppress("VARIABLE_IN_SINGLETON_WITHOUT_THREAD_LOCAL")
+    var cachedList = WordList()
```

Comment:
that would break the actual purpose of it being a `var`:

```
   var cachedList = WordList()

    fun getCachedWords(languageCode: String): List<String> {
        if (cachedList.languageCode != languageCode) {
            cachedList = WordList(languageCode)
        }
        return cachedList.words
    }
```

... I think I removed `internal` by accident. I'll fix that. However it could also be made `private`. It's not used anywhere else.

Reply:
also the suppress isn't technically needed. it will print a bunch of `info` level logs while building however:
`With old Native GC, variable in singleton without @ThreadLocal can't be changed after initialization`

Which is just annoying considering kotlin uses the new GC by default now

Reply:
Thanks—reviewing just on GitHub hides some of the important context!